config/ols-config/ols-config.yaml

```diff
@@ -81,7 +95,7 @@ ontologies:
     oboSlims: false
     ontology_purl : file:/mnt/ontologies/mondo-rare.owl
   - id: mondo
-    preferredPrefix: MONDO
+    preferredPrefix: MONDO_DEV
```

Comment:
in OLS, does this change the term IRIs?

Reply:
No, this is just used on the ontologies overview page.